angular/build/config/vendor.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
So if we rm -rf angular/build then we lose stuff?  I'm suggesting that there is a convention around build being a directory of output and my expectations were that all of it's content would be generated.

Maybe we can put the config outside of build?